libbeat/esleg/eslegclient/bulkapi.go

```diff
@@ -81,14 +81,22 @@ func (conn *Connection) Bulk(
 		return 0, nil, err
 	}
 
+	if conn.BulkResponseFiltering {
+		filteredFields := "errors,items.*.error,items.*.status"
+		if len(params) == 0 {
+			params = map[string]string{"filter_path": filteredFields}
+		} else {
+			params["filter_path"] = filteredFields
+		}
+	}
```

Comment:
The `Bulk` method here is rather generic.  As such I'm not in favor of hardcoding which paths to filter if the connection-level `BulkResponseFiltering` flag is enabled.  Why not just directly setup the `params` argument (fourth one) with the `filter_path` and filtered fields over here? https://github.com/elastic/beats/blob/3d0e8af41a1c02703f1f1d790d91594f201408ec/libbeat/outputs/elasticsearch/client.go#L248

Reply:
Thanks for the suggestion, I did not know that this was generic, looking at some references I did not see that, but you suggestion seems "safer"

Reply:
see 810c140